tools/releasing/collect_license_files.sh

```diff
@@ -57,4 +57,9 @@ LICENSES="${DST}/licenses"
 [ -f "${LICENSES}" ] && rm -r "${LICENSES}"
 find "${TMP}" -name "licenses" -type d -exec cp -r -- "{}" "${DST}" \;
 
+# Search the binary distribution directory and collect those license files that
+# not bundled in any jars.
+find "${SRC}/flink-dist/target" -name "LICENSE.*" -type f \
```

Comment:
```suggestion
find "${SRC}" -name "LICENSE.*" -type f \
```
`SRC` already points to the desired directory, see `create_binary_release.sh#78`